### ISSUE TITLE
Add a way to list packages by prefix

### ISSUE THREAD

**Issue by broady** [golang/gddo#399]
https://godoc.org/-/subrepo has a special index to display all the `golang.org/x/...` subrepos.

It would be nice to have this for any package path prefix. For example, to display all of the `google.golang.org/...` repos.

**Comment by shurcooL** [comment]
That does sound nice. Some queries of that nature may return an exorbitant amount of packages without pagination. Still, it will be less than the page that displays _all packages_ known to godoc.org.

@broady, in addition to `google.golang.org/...`, would this feature request would also include queries like `github.com/username/...` and `github.com/...`?

**Comment by broady** [comment]
Exactly. We should paginate. That other page that lists all packages should
be paginated, anyway!
